WordPressAuthenticator/Signin/LoginPrologueSignupMethodViewController.swift

```diff
@@ -14,6 +14,9 @@ class LoginPrologueSignupMethodViewController: NUXViewController {
     open var googleTapped: (() -> Void)?
     open var appleTapped: (() -> Void)?
 
+    /// The big transparent (dismiss) button behind the buttons
+    @IBOutlet weak var dismissButton: UIButton!
```

Comment:
nit: could this be private?

Reply:
Fixed in 59cc2ac0784a567da22061deb60023f310d20f44.